thoth/metrics_exporter/jobs.py

```diff
@@ -119,15 +118,29 @@ def get_solver_documents(solver_name: str = None):
     asyncio.set_event_loop(loop)
 
     try:
-        graph_db = GraphDatabase.create()
+        graph_db = GraphDatabase.create('janusgraph.test.thoth-station.ninja')
         graph_db.connect()
-    except aiohttp.client_exceptions.ClientConnectorError as excptn:
-        _LOGGER.error(excptn)
-        return
+        graphdb_connection_error_status.set(0)
+
+        # Optimized query
+        #number_solver_documents = graph_db.get_solver_documents_count()
+
+        # Non-Optimized query
+        number_solver_documents = asyncio.get_event_loop().run_until_complete(graph_db.
+        g.E()
+            .has("__label__", "solved")
+            .valueMap()
+            .select("solver_document_id")
+            .dedup()
+            .count()
+            .next())
```

Comment:
Feel free to directly supply this to thoth-storages not to database-specific queries split in the project.